openapi-cli/src/test/resources/generators/client/swagger/text_request_payload.yaml

```diff
@@ -0,0 +1,32 @@
+openapi: 3.0.1
+info:
+  title: Swagger Petstore
+  description: 'This is a sample server Petstore server.'
+  version: 1.0.0
+servers:
+  - url: https://petstore.swagger.io/v2
+paths:
+  /pets:
+    post:
+      summary: Add pets
+      operationId: addPets
+      requestBody:
+        content:
+          text/json:
+            schema:
+              $ref: '#/components/schemas/Pet'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+components:
+  schemas:
+    Pet:
+      type: object
+      properties:
+        Name:
+          type: string
+
```

Comment:
Shall we remove this extra new line?

Reply:
Removed in https://github.com/ballerina-platform/ballerina-openapi/pull/692/commits/1151944387857250e3b9f366f859863fac787fd6